addons/mail/static/src/discuss/web/partner_avatar_card/partner_avatar_card_popover.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module */
+import { AvatarCardPopover } from "@mail/discuss/web/avatar_card/avatar_card_popover";
+/** @type {ReturnType<import("@mail/utils/common/misc").rpcWithEnv>} */
+let rpc;
+import { rpcWithEnv } from "@mail/utils/common/misc";
+
+export class PartnerAvatarCardPopover extends AvatarCardPopover {
```

Comment:
I don't really get why we need to introduce yet another extension of `AvatarCardPopover`. We could just enrich the original one to handle different types of persona? A patch already exists for the hr module that displays employee instead of users. I'd say we should take this opportunity to provide proper support: `res.users`/`res.partner`/`hr.employee`. I feel like getting data should be as simple as:

```js
orm.searchRead(this.model, [["id, "=", this.props.id]], this.fieldNames)
```